Gemfile.lock

```diff
@@ -65,20 +65,21 @@ GEM
     diff-lcs (1.1.3)
     equalizer (0.0.11)
     erubis (2.7.0)
-    ethon (0.9.1)
+    ethon (0.10.1)
```

Comment:
These dependencies changes should be fine, but in general, unless required, let's keep the versions locked as-is.